package.json

```diff
@@ -30,6 +30,8 @@
     "test:creation-code": "hardhat test test-e2e/creationCodeTest.ts",
     "blockscout:verify": "hardhat run ./scripts/orbitVerifyOnBlockscout.ts",
     "deploy:usdc-token-bridge": "hardhat run scripts/usdc-bridge-deployment/deployUsdcBridge.ts",
+    "verify:usdc-token-bridge": "hardhat run scripts/usdc-bridge-deployment/verifyUsdcBridge.ts",
+    "verify:l1-usdc-token-bridge": "hardhat run scripts/usdc-bridge-deployment/verifyParentUsdcBridge.ts",
```

Comment:
Added to verify the contracts on the parent chain although they are likely to be already verified by bytecode match